dev/NumberBox/NumberBox.xaml

```diff
@@ -455,14 +560,14 @@
                             Grid.Column="1"
                             Style="{StaticResource DeleteButtonStyle}"
                             BorderThickness="{TemplateBinding BorderThickness}"
-                            contract7Present:CornerRadius="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=CornerRadius, Converter={StaticResource RightCornerRadiusFilterConverter}}"
+                            contract7Present:CornerRadius="{TemplateBinding CornerRadius}"
                             Padding="{ThemeResource HelperButtonThemePadding}"
                             IsTabStop="False"
                             Visibility="Collapsed"
                             AutomationProperties.AccessibilityView="Raw"
                             FontSize="{TemplateBinding FontSize}"
                             HorizontalAlignment="Right"
-                            MinWidth="34"
+                            MinWidth="40"
```

Comment:
Do we have to hardcode this number?

Reply:
It's hardcoded in the TextBox template, of which this is a copy.